### ISSUE TITLE
No effect in Single Pass Instanced

### ISSUE THREAD

**Issue by skyworxx** [Unity-Technologies/PostProcessing#362]
##### What happened?
The whole post processing stack seems to have no effect when running with SPS Instanced

##### Post-processing stack version (v1, v2)?
v2 (4a980688fd993bf6a410f67b09321a3ae2d3a65f)

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
2017.2.0p1-MRTP4

**Comment by robbiesri** [comment]
It's not configured yet.  Work on that will start Real Soon™

**Comment by robbiesri** [comment]
@brandon-unity3d will be taking over work for this 👯‍♂️

**Comment by bassmit** [comment]
Hope this gets fixed soon, would really like to be able to use the post processing stack, but we can't live without instancing, keep up the good work!

**Comment by adamkvd** [comment]
Any update on this? Would really appreciate single-pass instancing support for the PostProcessing stack V2 🙂

**Comment by robbiesri** [comment]
I ended up picking this back up. Locally, I had most of the base stereo instancing infrastructure working. I also decided to start the work for Android multi-view, which shares most of the architecture.  I ran into problems deploying to a Pixel, but I think the bugs weren't due to any of the stuff I was working on (hard to prove, of course).  

Anyway, once the core infrastructure is working, each update shouldn't be too hard to update on an individual basis.

**Comment by 01GOD** [comment]
Make sure it works on both viewports. When building to Android using XR and single pass stereo (and possibly other platforms) only one viewport renders and with no fx.

**Comment by owlboy** [comment]
Really looking forward to this working soon.

**Comment by 01GOD** [comment]
Well, at least can hope that Unity stopped trying to outsource to india because of this and similar hassles. 

This thread is a sadly hilarious commentary on why software companies SHOULDN'T EVER outsource to india and similar nations NOR import scammers from those nations.
 
"ravi" said he was "picking this back up" 3 months ago and then the next post is "ravi" removed this assignment 3 months ago. 

Obviously there were some brownie points to be scored by pretending to be doing something... and then dropping the project when the other employees looked away. 

The amount of cheating in schools and universities in india, china, and so on is seeming much too far beyond the capability of most in naive Western nations to understand. 

Basically "ravi" has proven everything I said months ago IS COMPLETELY TRUE.

**Comment by bassmit** [comment]
01GOD, I know it can be frustrating waiting for software to be developed
while it seems from the outside nothing is happening. You have no idea
about workloads and priorities though. Please re-read your email, it makes
you look like an angry biggot, Im sure that is not your intention.

Bas

**Comment by 01GOD** [comment]
@bassmit and any with similar unwritten statements

I highly recommend learning the difference between cultures, nations, and "races"...regardless of how much some minorities in Western nations have benefited from making the majority paranoid by somehow getting those terms mixed together in Western media. If a human is too heavily programmed by propaganda to admit blatant scientifically observable differences in genetics among various ethnic groups, then the human must still absolutely be capable of seeing differences in nations and cultures. Otherwise it would be an astounding feat for such a human to have had enough intelligence to operate a computer and post replies on this forum. 
I have resided around the World for as long as I remember now and have still occasionally been amazed at how VASTLY different and incompatible some nations and cultures really are. 
My statements were based on experiences, news reports, and statements directly from mouths of natives of those nations. 

I am a scientist and this is a scientific forum. The scientific method is based on observation...not rewriting facts to seem like a nice guy. 

Always remember to pause for a moment and recognize when a reply is simply a pawn-like expression of a media programmed algorithm for starting a tit-for-tat competition to see who can appear least "racist" for peer group points. 

Since somebody did use the "r" word, it's also worth noting that india is statistically the second most "racist" nation in the World...second only to jordan. Better to take that crusade there where it truly belongs. Have a look at how african students there are treated.

**Comment by kwi-dk** [comment]
01GOD, you were asked to change your behavior in #509, but evidently to no avail. I thus have no other option than to ban you.
